mysite/missions/templates/missions/shell/ls.html

```diff
@@ -0,0 +1,90 @@
+{% extends 'missions/shell/base.html' %}
+{% comment %}
+# This file is part of OpenHatch.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% block mission_main %}
+<div class='submodule fat'>
+    <div class='head'>
+      <h3>Using ls for listing directory contents</h3>
+    </div>
+    <div class="body">
+
+    <h3>ls: List directory contents</h3>
+    <p><b>1. List files using <kbd class="kbd">ls</kbd> with no options:</b></p>
+
+    <code class="code">$ ls</code>
+
+    <p><kbd class="kbd">ls</kbd> command with no option will list files and
+    directories in bare format and you won't be able to view details like file
+    types, size, modification date, permission etc.</p>
+
+    <p><b>2. List files using <kbd class="kbd">-l</kbd> option:</b></p>
+
+    <code class="code">$ ls -l</code>
+
+    <p><kbd class="kbd">ls</kbd> command with option <kbd class="kbd">-l</kbd> i.e.
+    <kbd class="kbd">ls -a</kbd> will list files and directories along with their
+    size, modified date and time, owner of the file and it's permission.
+    Something like:</p>
+    <pre>
+    <kbd class="kbd">drwxr-xr-x   2 root root  4096 Oct 11 00:20 bin</kbd>
+    </pre>
+
+    <p><b>3. View hidden files:</b></p>
+
+    <code class="code">$ ls -a</code>
+
+    <p><kbd class="kbd">ls</kbd> command with option <kbd class="kbd">-a</kbd>
+    i.e. <kbd class="kbd">ls -a</kbd> will list all the files including hidden
+    files starting with '.' (hidden file's name starts with '.')</p>
+
+    <p>There are many other options available. For more options, type
+    <kbd class="kbd">ls --help</kbd> in your shell and you'll get a list of
+    available options along with a short description for each option.</p>
```

Comment:
@vipul-sharma20 zsh does not install a help function be default. It may be good to mention that on zsh, you may type "man ls" to get options.